sysinit/coqargs.ml

```diff
@@ -317,8 +317,8 @@ let parse_args ~usage ~init arglist : t * string list =
       add_set_option oval ["Mangle"; "Names"; "Prefix"] (OptionSet(Some(next ())))
 
     |"-profile-ltac-cutoff" ->
-      Flags.profile_ltac_cutoff := get_float ~opt (next ());
```

Comment:
ltac profiling used to be in ltac plugin, since plugins can't use extra command line options the flag needed to be in some statically linked code